example/lib/main.dart

```diff
@@ -25,19 +23,16 @@ class _SearchBarDemoHomeState extends State<SearchBarDemoHome> {
   final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
 
   AppBar buildAppBar(BuildContext context) {
-    return new AppBar(
-        title: new Text('Search Bar Demo'),
-        actions: [searchBar.getSearchAction(context)]);
+    return new AppBar(title: new Text('Search Bar Demo'), actions: [searchBar.getSearchAction(context)]);
   }
 
   void onSubmitted(String value) {
-    setState(() => _scaffoldKey.currentState
-        .showSnackBar(new SnackBar(content: new Text('You wrote $value!'))));
+    setState(() => _scaffoldKey.currentState.showSnackBar(new SnackBar(content: new Text('You wrote $value!'))));
   }
 
   _SearchBarDemoHomeState() {
     searchBar = new SearchBar(
-        inBar: false,
```

Comment:
Is there any particular reason this is the default now, or just a testing thing?

Reply:
Yeah my bad, it was a testing thinghie :)